stripe/src/main/java/com/stripe/android/view/AddAddressWidget.java

```diff
@@ -0,0 +1,350 @@
+package com.stripe.android.view;
+
+import android.content.Context;
+import android.support.design.widget.TextInputLayout;
+import android.util.AttributeSet;
+import android.view.View;
+import android.widget.AdapterView;
+import android.widget.LinearLayout;
+import android.widget.Spinner;
+
+import com.stripe.android.R;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Locale;
+
+/**
+ * An add address widget using the support design library's {@link TextInputLayout}
```

Comment:
Nit: comment should probably focus on what you can do with the widget, rather than the design -- something like 
"A widget used to collect address data from a user"
In the case of the CardMultilineWidget, the comment about design is used to contrast it with CardInputWidget.